hms-app/src/components/card-details/IdeaDetails.tsx

```diff
@@ -132,6 +132,212 @@ export default function IdeaDetails(props: IProps) {
+                ideaData.owner?.firstName,
+                ideaData.owner?.lastName
+                )}
+            </Avatar>
+            <Badge size='sm'>
+                {ideaData.owner?.firstName} {ideaData.owner?.lastName}
+            </Badge>
+            </Stack>
+            <Text className={classes.title}>
+            {ideaData.title?.slice(0, MAX_TITLE_LENGTH)}
+            {ideaData.title?.length > MAX_TITLE_LENGTH ? '...' : ''}
+            </Text>
+        </Group> 
+    )
+        
+  }
+    
+  const ideaDescription = () => {
+      return (
+          <Text className={classes.text}> 
+              {ideaData.description}
+          </Text>
+      )
+  }
+
+  const ideaProblem = () => {
+      return (
+          <Card.Section className={classes.borderSection}>
+              <Text className={classes.label}>Problem</Text>
+              <Text className={classes.text}>{ideaData.problem}</Text>
+          </Card.Section>
+      )
+      
+  }
+
+  const ideaGoal = () => {
+      return (
+          <Card.Section className={classes.borderSection}>
+              <Text className={classes.label}>Goal</Text>
+              <Text className={classes.text}>{ideaData.goal}</Text>
+          </Card.Section>
+      )
+      
+  }
+
+  const ideaCategory = () => {
+      return (
+          <Card.Section className={classes.borderSection}>
+              <Text className={classes.label}>Category</Text>
+              <Tooltip
+                  multiline
+                  width={220}
+                  transition='fade'
+                  transitionDuration={200}
+                  color='gray'
+                  label={categoryData.description}
+              >
+                  <Badge
+                  color={theme.colorScheme === 'dark' ? 'dark' : 'gray'}
+                  key={ideaData.category?.id}
+                  >
+                  {ideaData.category?.title}
+                  </Badge>
+              </Tooltip>
+          </Card.Section>
+      )
+      
+  }
+
+  const ideaRequiredSkills = () => {
+      return (
+          <div>
+              <Text className={classes.label}>Skills required</Text>
+              <Group spacing={7} mt={5}>
+                  {ideaData.requiredSkills?.map((skill) => (
+                  <Tooltip
+                      multiline
+                      width={220}
+                      transition='fade'
+                      transitionDuration={200}
+                      color='gray'
+                      label={getSkillDescription(skill.id)}
+                      key={skill.id}
+                  >
+                      <Badge
+                      color={theme.colorScheme === 'dark' ? 'dark' : 'gray'}
+                      key={skill.id}
+                      >
+                      {skill.name}
+                      </Badge>
+                  </Tooltip>
+                  ))}
+              </Group>
+          </div>
+          )
+  }
+
+  const ideaSkills = ()  => {
```

Comment:
this const feels unnessasary 
you could replace the "div" in required skills const with "card.section" and would save some lines of code